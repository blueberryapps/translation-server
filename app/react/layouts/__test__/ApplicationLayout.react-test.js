import React from 'react';
import renderer from 'react-test-renderer';

import Layout from '../ApplicationLayout.react';

it('renders correctly', () => {
  const tree = renderer.create(
    <Layout>Homepage</Layout>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
