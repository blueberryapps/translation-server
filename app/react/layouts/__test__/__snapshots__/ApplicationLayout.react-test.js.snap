// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "overflowX": "hidden",
      }
    }
  >
    Homepage
  </div>
</div>
`;
